_src/assets/SVG/facebook-square-social-logo-svgrepo-com.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
The folder SVG is no longer needed. When there's a file such as 'defs.svg' there's no need to keep the svg image files.